### PR TITLE
DOC: Make Travis-CI integration instructions more specific.

### PR DESCRIPTION
**Pull request by danielballan**
I created a new GitHub account and went through the Travis-CI instructions to
be sure that they would work for a normal new user without a pre-existing
Travis-CI account or any pre-existing settings or integrations.

Fortunately, the instructions were correct, but I have made them a little more
specific about exactly what a new user sees.

### CODE REVIEW

File: docs/source/ci.rst [hunk 1]
```diff
@@ -37,17 +37,22 @@ remove the line that does ``flake8`` style-checking.
...
+   ``https://travis-ci.org/YOUR_GITHUB_USERNAME/YOUR_REPO_NAME/builds``. The
+   list will currently be empty. You'll see construction cones.
+#. The next time you open a pull request or push a new commit to the master
+   branch, Travis-CI will kick off a new build, and that list will update. As a
+   quick check, you can push an empty commit to master:
+
+   .. code-block:: bash
+
+      git commit --allow-empty -m "CI: Test Travis-CI integration."
+      git push origin master
```
Comment (danielballan) [inline docs/source/ci.rst]: I'm curious if people think this is too dirty. This is what _I_ did to quickly check things are working, but I'm bit worried that it sets a bad example for people less versed in good practices. It is generally not good to throw empty commits at `master` for debugging purposes.... Should I remove this?

Comment (tacaswell) [reply docs/source/ci.rst]: eh, 50/50 on this.  I would just open a PR to test...

Comment (danielballan) [reply docs/source/ci.rst]: K. Force-pushed an update to remove this.

**Comment by danielballan** [comment]
Reviewed on Slack by @awalter-bnl. Merging to get this up.

**Pull request merged**